apps/vaporgui/SettingsParams.cpp

```diff
@@ -402,7 +402,7 @@ void SettingsParams::SetFidelityDefault2D(long lodDef, long refDef)
     SetValueLongVec(_fidelityDefault2DTag, "Set fidelity 2D default", val);
 }
 
-bool SettingsParams::_loadFromSettingsFile()
+int SettingsParams::LoadFromSettingsFile()
```

Comment:
Why was the return type change from bool to int? Our error policy states that if a function logs an error message (via MyBase::SetErrMsg) it should return an int (or NULL pointer) to indicate failure. If it does not log a message it should return a boolean.

Reply:
Fixed.